packages/git-tools/scripts/git-valu-push-production

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -eu
+set -o pipefail
+
+if [ "$(git rev-parse --abbrev-ref HEAD)" != "master" ]; then
+  echo "You can deploy to production only from the local master branch"
+  exit 1
+fi
+
```

Comment:
Uncommitted changes are not deployed. Usually it is not intentional to leave changes not deployed. If user really wants this they can manually stash them.

```suggestion

if [ "$(git status --porcelain)" != "" ]; then
  echo "Dirty. Commit changes before deploy"
  exit 1
fi

```